packages/mapviewer/src/modules/drawing/components/DrawingToolboxButton.vue

```diff
@@ -59,9 +59,8 @@ function setDrawingMode() {
         <small
             v-if="!isPhoneMode"
             class="d-sm-block"
-        >{{
-            t(`draw_${drawingMode.toLowerCase()}`)
-        }}</small>
+            >{{ t(`draw_${drawingMode.toLowerCase()}`) }}</small
+        >
```

Comment:
Same here but with the <small> tag, I usually tend to just line break after the declaration...